l2-frontend/src/pages/Profiles.vue

```diff
@@ -1001,6 +1050,10 @@ export default {
         }
       }
     },
+    async getAnalyzer() {
+      const list = await this.$api('manage-analyze');
```

Comment:
Вообще этот урл в целом непонятный
Выше я про него писал, но в целом он должен отражать суть
Что то вроде
```suggestion
      const list = await this.$api('manage-analyzers/all-analyzers');
```